src/main/kotlin/com/project/geo/controller/GeoController.kt

```diff
@@ -0,0 +1,28 @@
+package com.project.geo.controller
+
+
+import com.project.geo.service.StreetGeometryService
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RequestParam
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@RequestMapping("api")
+class GeoController(
+    private val streetGeometryService: StreetGeometryService
+) {
+
+    @GetMapping("street/{address}")
+    fun streetGeometry(@PathVariable address: String,
+                       @RequestParam("south_west") southWest: List<Double>,
```

Comment:
Is there any particular reason to receive it in this case?

Reply:
What do you mean?